setuptools_rust/build.py

```diff
@@ -37,9 +37,9 @@
 
 
 try:
-    from wheel.bdist_wheel import bdist_wheel as CommandBdistWheel
-except ImportError:  # wheel installation might be deferred in PEP 517
-    from setuptools import Command as CommandBdistWheel
+    from setuptools.command.bdist_wheel import bdist_wheel as CommandBdistWheel
+except ImportError:  # old version of setuptools
+    from setuptools import Command as CommandBdistWheel  # type: ignore[assignment]
```

Comment:
You should instead try all 3 imports...

Reply:
No problems, I can do that. I skipped the `wheel` import in this particular case because it is used for type checking.

Reply:
Implemented in [2e1d65b](https://github.com/PyO3/setuptools-rust/pull/463/commits/2e1d65baa206f1aaea6631a47b788631b6f5a1bc).